### PR TITLE
Add Copay.app v1.9.4

### PR DESCRIPTION
**Pull request by ghost**
Copay is a open source bitcoin wallet developed mainly by bitpay. I'm not affiliated with them, but if this PR is accepted I will let them know that the app is available from homebrew

### CODE REVIEW

**Comment by adidalal** [comment]
Please use a versioned URL and add in an appcast: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

See the linked electron Cask or ask here if you have any questions!

File: Casks/copay.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'copay' do
+  version '1.9.4'
+  sha256 'c55625e36c5c20c6b5d7978df197a8691b9d43790530f62a157c34c1090dcf82'
+
+  url 'https://github.com/bitpay/copay/releases/download/v1.9.4/Copay-osx.dmg'
+  name 'Copay'
+  homepage 'https://copay.io'
+  license :oss # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/copay.rb]: You need to, as the comment says, remove the comment.

File: Casks/copay.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'copay' do
+  version '1.9.4'
+  sha256 'c55625e36c5c20c6b5d7978df197a8691b9d43790530f62a157c34c1090dcf82'
+
+  url 'https://github.com/bitpay/copay/releases/download/v1.9.4/Copay-osx.dmg'
```
Comment (vitorgalvao) [inline Casks/copay.rb]: `url "https://github.com/bitpay/copay/releases/download/v#{version}/Copay-osx.dmg"`.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by bektar** [comment]
Hopefully i just did… let me know if i messed it up :)

**Comment by vitorgalvao** [comment]
> Hopefully i just did… let me know if i messed it up :)

You haven’t. Most people do so on their first try. Will fix the `url`, though.

File: Casks/copay.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/copay.rb]: Needs a `# github.com/bitpay/copay was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 239e4b569db0b19f01c495c6a31bb65faa5edd13. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by bektar** [comment]
Aw, dammit. Oh well, thanks for the help

**Pull request closed without merging**